cpp/arcticdb/util/test/test_storage_lock.cpp

```diff
@@ -16,6 +16,7 @@ using namespace arcticdb;
 using namespace folly;
 
 TEST(StorageLock, SingleThreaded) {
+    SKIP_MAC("StorageLock is not supported");
```

Comment:
If possible, could you indicate why `StorageLock` is not supported on Mac? Is it due to mutex implementation on MacOS?

Reply:
it's  segfaulting for almost all StorageLock tests